packages/gatsby-remark-shiki-twoslash/src/dom.ts

```diff
@@ -63,8 +63,10 @@ export const setupTwoslashHovers = () => {
 
     // Use a textarea to un-htmlencode for presenting to the user
     var txt = document.createElement('textarea')
-    txt.innerHTML = message
-    tooltip.textContent = txt.value
+    txt.innerHTML = message;
+    tooltip.innerHTML = txt.value
+    // remove shiki wrapper element
+    tooltip.innerHTML = tooltip.querySelector('code')?.innerHTML!;
```

Comment:
Because we're using the same highlighter logic, the lsp value will be wrapped with `pre` and additional `div` element.

```html
<pre class="shiki twoslash">
  <div class="language-id">
  <code>actual definition here</code>
</pre>
```

Because we only need to content of the `code` element I'm assuming this is safe to do